Libraries/test/EventsTests.Shared/dynamodb-event.json

```diff
@@ -41,10 +54,14 @@
       },
       {
          "eventID":"f07f8ca4b0b26cb9c4e5e77e42f274ee",
-         "eventName":"INSERT",
```

Comment:
Updated this record too to check for `UserIdentity` (it's only available in `REMOVE` events).